execline-2.0.1.1/debian/copyright

```diff
@@ -0,0 +1,13 @@
+Copyright (c) 2011-2014 Laurent Bercot <ska-skaware@skarnet.org>
```

Comment:
@bukzor If you're wanting to get this into debian, there's a specific format for `debian/copyright`. Here's an example (link to the format is at the top as well): https://github.com/chriskuehl/pre-commit/blob/debian/debian/copyright

(obviously not a blocker)